tensorflow/contrib/tensorrt/kernels/trt_engine_op.cc

```diff
@@ -64,10 +74,15 @@ void TRTEngineOp::Compute(OpKernelContext* context) {
     const TensorShape& input_shape = input_tensor.shape();
     if (i == 0) {
       num_batch = input_shape.dim_size(0);
+      if (num_batch > trt_engine_ptr_->getMaxBatchSize())
```

Comment:
We have to discuss this in more detail. It might not be good idea to split the batches. We might also think of creating multiple engines running on part of the batch and merging the output results afterwards. But that would be similar implementing a mini-framework in op and not sure a good idea.